apps/server/src/modules/video-conference/uc/video-conference-info.uc.ts

```diff
@@ -87,7 +71,7 @@ export class VideoConferenceInfoUc {
 				scope.id,
 				scope.scope
 			);
-			options = vcDO.options;
+			options = { ...vcDO.options };
```

Comment:
does that change the behaviour for the options given by the modal for VC start or why was that necessary?

Reply:
Was also a linter warning fix. It recommended to use object destructuring and that was the only way to write that. Actually I'm not quite happy with the notation also but I think it should do the job 🤷‍♂️